### ISSUE TITLE
quick question for user interface

### ISSUE THREAD

**Issue by elf7979** [Marksdo/Whisper#13]
this is my whisper mate user interaface

<img width="1094" alt="스크린샷 2023-12-12 오전 10 35 41" src="https://github.com/Marksdo/Whisper/assets/15778931/40b4e4b3-3b58-40ad-9523-8f3d220b5d91">

My mac os is Monterey.

I notice other people have different user interface.
Do I need to upgraded to Ventura to see that UI? 
Also, I can't find the way to use deepgram engine with the current inteface.
If I upgraded to Ventura, I can find the way to use deepgram engine?

**Comment by Marksdo** [comment]
Yes. macOS 12 and macOS 13 use difference implement UI (cause Table has
issue in macOS12 Monterey)

Deepgram option is here
[image: image.png]
You need go to deepgram website register an free $200 api token. You don't
need to upgrade to Ventura

**Comment by Marksdo** [comment]
Yes. macOS 12 and macOS 13 use difference implement UI (cause Table has
issue in macOS12 Monterey)

Deepgram option is here
<img width="388" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/d6479448-4129-4449-a915-3cb5578faf48">

You need go to deepgram website register an free $200 api token. You don't
need to upgrade to Ventura

**Comment by Marksdo** [comment]
If you want to use Realtime transcribe with DeepGram, you need to upgrade to Ventura. Realtime record feature is only available in Ventura (macOS 13)
<img width="415" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/b8930a80-2da3-420c-a13c-60c8a29477a2">

**Comment by elf7979** [comment]
> Yes. macOS 12 and macOS 13 use difference implement UI (cause Table has issue in macOS12 Monterey)
> 
> Deepgram option is here <img alt="image" width="388" src="https://private-user-images.githubusercontent.com/117704549/289718974-d6479448-4129-4449-a915-3cb5578faf48.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzNjk2MzQsIm5iZiI6MTcwMjM2OTMzNCwicGF0aCI6Ii8xMTc3MDQ1NDkvMjg5NzE4OTc0LWQ2NDc5NDQ4LTQxMjktNDQ0OS1hOTE1LTNjYjU1NzhmYWY0OC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMlQwODIyMTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1lN2JiMWFhZmFiZWY5NjliZDFkYzZlY2M1M2QxZTEwM2RmM2ZlZDc1MGM3Yzc5ZmEzNmFjMDcyZWI3Y2Q4Zjg2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.dTy5RQlaS2vaZPAoMWgjz8Q4kreHqBRF3I-zC2tnfqo">
> 
> You need go to deepgram website register an free $200 api token. You don't need to upgrade to Ventura

I can't find that engine menu on my whisper mate pro. 
That is why I am asking whether Monterey UI doesn't have engine menu

**Comment by elf7979** [comment]
I am truly sorry to consume your time.
I already updated whispermate pro all the time and I am using the latest version(V5.4.5) in Monterey.

And this is my UI

<img width="799" alt="스크린샷 2023-12-12 오후 6 26 01" src="https://github.com/Marksdo/Whisper/assets/15778931/c24786c6-56b8-40f0-b44b-dfd6f31c2f60">

I can't bring up that side window from your image including record option

**Comment by Marksdo** [comment]
before process project click 
<img width="627" alt="image" src="https://github.com/Marksdo/Whisper/assets/117704549/579415de-0ab1-4e5e-a40c-d34d9e87bbb7">

**Comment by elf7979** [comment]
> before process project click <img alt="image" width="627" src="https://private-user-images.githubusercontent.com/117704549/289812322-579415de-0ab1-4e5e-a40c-d34d9e87bbb7.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDIzODA4MzcsIm5iZiI6MTcwMjM4MDUzNywicGF0aCI6Ii8xMTc3MDQ1NDkvMjg5ODEyMzIyLTU3OTQxNWRlLTBhYjEtNGU1ZS1hNDBjLWQzNGQ5ZTg3YmJiNy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBSVdOSllBWDRDU1ZFSDUzQSUyRjIwMjMxMjEyJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDIzMTIxMlQxMTI4NTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hMzliMmQyYzdlMTMwMWY2MzA3YzJiMWE4MDNmNjkyNmZkMDFhYWJlYzlkMzg2OTVmNmExZjQwNzljYmFiYWFjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.B2Zn6AMs5nvNp_MDCMxqd6NL_33cKB4UQHFVMnwa3kU">

I always used local model. So I didn't even look at that icon.
Thanks a lot. I appreciate your time

**Comment by Marksdo** [comment]
🤝
